### PR TITLE
Feature/android background push

### PR DESCRIPTION
**Pull request by gwyddion39**

### CODE REVIEW

**Comment by gwyddion39** [comment]
@egorpushkin PTAL

File: samples/EnRouteDemo/EnRouteDemo.cs [hunk 1]
```diff
@@ -10,15 +10,9 @@ namespace EnRouteDemo
...
-        private GEnRouteFactory _enRouteFactory;
-
-        private GEnRouteManager _enRouteManager;
-
         public App (GEnRouteFactory enRouteFactory)
         {
-            _enRouteFactory = enRouteFactory;
-
-            _enRouteManager = _enRouteFactory.createEnRouteManager();
+			EnRouteManagerWrapper.Instance.create(enRouteFactory);
```
Comment (egorpushkin) [inline samples/EnRouteDemo/EnRouteDemo.cs]: Tabs somehow sneaked in here.

Comment (gwyddion39) [inline samples/EnRouteDemo/EnRouteDemo.cs]: Fixed

**Comment by egorpushkin** [comment]
LGTM

**Pull request merged**